Gemfile

```diff
@@ -32,6 +32,7 @@ end
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'pry-rails'
+  gem 'rspec-json_expectations'
```

Comment:
I am not sure that we should use this gem. It has not been updated for 4 years. This could cause technical debt in the future.

Reply:
This PR is to revert to merge. In the next PR, include_json matcher needs an alternative then; two tests need changing.